lib/msal-common/src/authority/AadAuthority.ts

```diff
@@ -37,7 +37,7 @@ export class AadAuthority extends Authority {
 
         // for custom domains in AAD where we query the service for the Instance discovery
         const response = await this.networkInterface.sendGetRequestAsync<OpenIdConfigResponse>(this.aadInstanceDiscoveryEndpointUrl);
-        return response.tenant_discovery_endpoint;
+        return response.body.tenant_discovery_endpoint;
```

Comment:
Doesn't this break fetch for msal-browser?

Reply:
The response from fetch is wrapped in a NetworkResponse object, defined in msal-common. This is part of the throttling PR